src/python/pants/jvm/resolve/jvm_tool_test.py

```diff
@@ -83,16 +84,8 @@ def test_jvm_tool_base_extracts_correct_coordinates() -> None:
         }
     )
     lockfile_request = rule_runner.request(JvmLockfileRequest, [MockJvmToolLockfileSentinel()])
-    assert sorted(lockfile_request.artifact_inputs) == [
-        "//:junit_junit",
```

Comment:
One side-effect of this assert here is that we validated that both Maven-style and Address-style artifact specifications were tests. Are we doing this anywhere else?

Reply:
Oh sorry, I had on auto-merge. Will make sure the followup PR has tests for this.